benchmarks/index.html

```diff
@@ -15,7 +22,32 @@
   margin: 20px auto;
   font-family: Helvetica, Arial, "sans-serif";
 }
+
+text {
+    font: 12px sans-serif;
+}
+svg {
+    display: block;
+}
+
+.gallery {
+  display: none;
+}
+
+.gallery, svg {
+  height: 300px;
+  width: 100%;
+}
+
+.title {
+  font-weight: bold;
+  font-size: 16px;
+}
+
 </style>
+<script>
+  window.loadCount = 40;
```

Comment:
let's just keep it at 251 for now. I agree with having it defined as a global variable though